plutus-tx-plugin/test/Budget/notElemCheap.pir-readable.golden

```diff
@@ -41,9 +38,4 @@ let
         (\(ds : integer) (ds : List integer) -> Cons {integer} ds ds)
         (Nil {integer})
 in
-Bool_match
-  (go x)
-  {all dead. Bool}
-  (/\dead -> False)
-  (/\dead -> True)
-  {all dead. dead}
```

Comment:
Was it GHC who optimized this for us or is our own optimizer now that clever as well?